src/main/java/bio/terra/flight/study/create/CreateStudyAuthzResource.java

```diff
@@ -13,26 +16,37 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import java.util.List;
 import java.util.UUID;
 
 public class CreateStudyAuthzResource implements Step {
+    private static Logger logger = LoggerFactory.getLogger(CreateStudyAuthzResource.class);
 
     private SamClientService sam;
-    public CreateStudyAuthzResource(SamClientService sam) {
+    private BigQueryPdao bigQueryPdao;
+    private StudyService studyService;
+
+    public CreateStudyAuthzResource(
+        SamClientService sam,
+        BigQueryPdao bigQueryPdao,
+        StudyService studyService) {
         this.sam = sam;
+        this.bigQueryPdao = bigQueryPdao;
+        this.studyService = studyService;
     }
 
-    private static Logger logger = LoggerFactory.getLogger(CreateStudyAuthzResource.class);
-
     @Override
     public StepResult doStep(FlightContext context) {
         FlightMap inputParameters = context.getInputParameters();
         AuthenticatedUserRequest userReq = inputParameters.get(
             JobMapKeys.USER_INFO.getKeyName(), AuthenticatedUserRequest.class);
         FlightMap workingMap = context.getWorkingMap();
         UUID studyId = workingMap.get("studyId", UUID.class);
+        Study study = studyService.retrieve(studyId);
         try {
-            sam.createStudyResource(userReq, studyId);
+            List<String> policyEmails = sam.createStudyResource(userReq, studyId);
```

Comment:
What is a policy Email?
Maybe a comment here on who is getting access to what would be useful.